client/my-sites/site-settings/form-general.jsx

```diff
@@ -433,20 +421,19 @@ module.exports = React.createClass( {
 						<SectionHeader label={ this.translate( 'Jetpack' ) }>
 							{ this.jetpackDisconnectOption() }
 							{ config.isEnabled( 'manage/option_sync_non_public_post_stati' )
-							  ? <Button
-							  	compact={ true }
-								onClick={ this.submitForm }
-								primary={ true }
-								type="submit"
-
-								disabled={ this.state.fetchingSettings || this.state.submittingForm }>
+								? <Button
```

Comment:
I wonder if a `<ConfigurationGate key="manage/option..."><Button ... /> </ConfigurationGate>` would make sense down the road...